modules/test/playwright/env/setUp.env.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {startAppServer, waitForStartedAppServer} from './appServerUtil.env';
+import {deployProjectClientExtensions, deployProjectDeployDir, deployProjectOSGiModules, updatePortalExtProperties} from './common.env';
+
+updatePortalExtProperties();
+
+deployProjectOSGiModules();
+
+deployProjectClientExtensions();
+
+deployProjectDeployDir();
+
+startAppServer();
+
+waitForStartedAppServer();
```

Comment:
@michaelhashimoto, dude, this looks so clean! 🎉  Explicit references is so nice to have!